src/controllers/comment.js

```diff
@@ -0,0 +1,50 @@
+const Controller = require('./controller');
+const CommentModel = require('../models/comment');
+
+class CommentController extends Controller {
+  constructor() {
+    super(CommentModel);
+  }
+
+  /**
+   * @param {string} id
+   */
+  async get(id) {
+    const comment = await this.model.Get(id);
+
+    if (!comment) {
+      console.log('comment not found');
+      return {};
+    }
+    return comment;
+  }
+  /**
+   *
+   * @param {string} body
+   * @returns {Promise<void>}
```

Comment:
this doesn't return void it returns an id right?

Reply:
If this is a question then no it does not return void I dont think anyways

Reply:
if the return isn't right you should fix or remove it

Reply:
everything returns an Id